test/conformance/service_test.go

```diff
@@ -259,12 +259,19 @@ func TestUpdateRevisionTemplateSpecMetadata(t *testing.T) {
 	}
 
 	logger.Info("Updating labels of the RevisionTemplateSpec for service %s", names.Service)
-	svc = reloadService(names.Service, clients, t)
-	svc.Spec.RunLatest.Configuration.RevisionTemplate.Labels = map[string]string{
-		"labelX": "abc",
-		"labelY": "def",
-	}
-	svc, err = clients.ServingClient.Services.Update(svc)
+	metadata := metav1.ObjectMeta{
+		Labels: map[string]string{
+			"labelX": "abc",
+			"labelY": "def",
+		},
+	}
+	svc, err = test.UpdateServiceRevisionTemplateMetadata(logger, clients, svc, metadata)
+	//labels := map[string]string{
+	//	"labelX": "abc",
+	//	"labelY": "def",
+	//}
+	//
+	//err = updateServiceWithLabels(clients, names, labels)
```

Comment:
I think you forgot to uncomment :)

Reply:
Actually I forgot to delete it :) The commented out code is replaced with the `metadata` variable and the call to `UpdateServiceRevisionTemplateMetadata`.